<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameClass</class>
 <widget class="QMainWindow" name="GameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/pattern.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/skribbl name/logo.gif);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="DrawingWidget" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>621</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>1041</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/clock/clock.gif);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pus numarul rundei</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentAction">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ce se intampla in joc</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chat">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>130</y>
      <width>211</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
	background:transparent;
    background-color: rgba(255, 255, 255, 0.9); /* Alb cu transparență pentru a lăsa să se întrevadă fundalul */
    color: rgb(0, 0, 0); /* Textul în culoarea neagră pentru citire ușoară */
    border: 1px solid rgb(255, 255, 255); /* Bordură albă clară */
    border-radius: 5px; /* Margini rotunjite */
    padding: 5px; /* Spațiere internă pentru text */
    font-size: 16px; /* Mărimea fontului adecvată */

}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoEditMenu|Qt::ImhNoTextHandles</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMesageButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>640</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background:transparent;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0); /* text negru */
    border-style: solid;
    border-width: 2px;
    border-radius: 10px; /* colțuri rotunjite */
    border-color: rgba(0, 0, 0, 150);
    padding: 5px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 200); /* mai puțin transparent când treci cu mouse-ul peste */
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 100); /* mai transparent când este apăsat */
}
</string>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>640</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background:transparent;
    background-color: rgba(255, 255, 255, 0.9); /* Alb cu transparență pentru a lăsa să se întrevadă fundalul */
    color: rgb(0, 0, 0); /* Textul în culoarea neagră pentru citire ușoară */
    border: 1px solid rgb(255, 255, 255); /* Bordură albă clară */
    border-radius: 5px; /* Margini rotunjite */
    padding: 5px; /* Spațiere internă pentru text */
    font-size: 16px; /* Mărimea fontului adecvată */

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>aici va fi linkul catre joc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/settings/settings.gif);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Chat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Player1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Player2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #f6f7fa, stop:1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default
 {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Albastru">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(0, 0, 255);
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Verde">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>670</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(0, 255, 0);
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Rosu">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(255, 0, 0);
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Orange">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: #FFA500;
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Purple">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>670</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: #800080;
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Brown">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>630</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
 {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: #A52A2A;
    min-width: 80px;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #dadbde, stop:1 #f6f7fa);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:default 
{
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>background</zorder>
   <zorder>label_5</zorder>
   <zorder>drawingArea</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>CurrentAction</zorder>
   <zorder>chat</zorder>
   <zorder>SendMesageButton</zorder>
   <zorder>label_9</zorder>
   <zorder>textEdit</zorder>
   <zorder>SettingsButton</zorder>
   <zorder>Chat</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>Clear</zorder>
   <zorder>Albastru</zorder>
   <zorder>Verde</zorder>
   <zorder>Rosu</zorder>
   <zorder>Orange</zorder>
   <zorder>Purple</zorder>
   <zorder>Brown</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingWidget</class>
   <extends>QLabel</extends>
   <header>DrawingWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="background.qrc"/>
  <include location="settings.qrc"/>
  <include location="clock.qrc"/>
  <include location="skribbl name.qrc"/>
 </resources>
 <connections/>
</ui>
